MishaKav/workflow-test
# format

This PR move the "serverless" option from organization level to user level.

Refs: [TS-9020](https://contrast.atlassian.net/browse/TS-9020), part of [TS-8753](https://contrast.atlassian.net/browse/TS-8753)

| Before                                    | After                                    |
| ----------------------------------------- | ---------------------------------------- |
| ![image](https://user-images.githubusercontent.com/289035/129388837-48b9b985-2e12-45d6-954e-3a78fedca30d.png) | ![image](https://user-images.githubusercontent.com/289035/129388727-f19673c1-f1e5-4471-a8ec-18757c68a84e.png)|


### Changes

- Add the ability to turn-on "serverless" for admins to individual users

### Testing

1. Open contrast-ui, check that you don't see a "serverless tab"
2. Navigate to "organization settings" > "users", check that you don't have "serverless column"
3. Check that you have `cloudnative` feature flag enabled ([How to Enable CloudNative in Teamserver](https://contrast.atlassian.net/wiki/spaces/CLOUDNATIVE/pages/1901103892/Enabling+Cloud+Native+in+Teamserver))
4. Login with superadmin, click on organization settings check the option "Enable Cloudnative" and save
![image](https://user-images.githubusercontent.com/289035/129469664-a3a5f54d-35bc-4ae3-9176-09308d19bc52.png)
5. Login with admin, navigate to "organization settings" > "users", check that you have "serverless column"
6. Turn-on "serverless switch" to current user, and see that you see "serverless tab"
![image](https://user-images.githubusercontent.com/289035/129469774-74b8a353-fdf9-4eac-b129-988162c831e6.png)



### Added e2e tests:

1. check that `serverless` tab is not showing by default
2. check that `serverless` tab is showing when user has an `serverless_enabled: true`
3. check that admin don't has "serverless switch" by default
4. check that admin has "serverless switch" if organization has `isCloudNativeEnabled: true`

### Pull request checklist

- [X] Code formatted
